server/checks/Approval.js

```diff
@@ -139,9 +138,12 @@ export default class Approval extends Check {
             return
           }
           // get approvals for pr
-          const opener = pull_request.user.login
+          const commits = await github.fetchPullRequestCommits(user, repo, number, token)
+          const lastCommitter = commits.length === 0 ?
+            null :
```

Comment:
I just tried and it's not, if the head of the compare branch is in the base branch, Github always marks the pull request as merged.